en-US/downloads/gui-locale-available.md

```diff
@@ -0,0 +1,117 @@
+---
+published: true
+layout: default
+title: Available GUI languages for FontForge
+---
+
+Available GUI languages for FontForge
+--------------
+
+<table class="table" border>
```

Comment:
You can write this table in markdown format using somehting like https://www.tablesgenerator.com/markdown_tables

Reply:
OK, I've changed the table into markdown.